src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+
+tableHead.addEventListener('click', onClick);
+
+function onClick(e) {
+  const th = e.target;
+
+  sortData(th.cellIndex, th.innerHTML);
+};
+
+function sortData(colIndex, colName) {
+  const rowsArr = Array.from(tableBody.rows);
+
+  switch (colName) {
+    case 'Age':
+      rowsArr.sort((rowOne, rowTwo) =>
+        rowOne.cells[colIndex].innerHTML
+          - rowTwo.cells[colIndex].innerHTML
+      );
+      break;
+
+    case 'Name':
+      rowsArr.sort((rowOne, rowTwo) =>
+        rowOne.cells[colIndex].innerHTML
+          .localeCompare(rowTwo.cells[colIndex].innerHTML)
+      );
+      break;
+
+    case 'Position':
+      rowsArr.sort((rowOne, rowTwo) =>
+        rowOne.cells[colIndex].innerHTML
+          .localeCompare(rowTwo.cells[colIndex].innerHTML)
+      );
+      break;
```

Comment:
Avoid code duplicate
```suggestion
    case 'Name':
    case 'Position':
      rowsArr.sort((rowOne, rowTwo) =>
        rowOne.cells[colIndex].innerHTML
          .localeCompare(rowTwo.cells[colIndex].innerHTML)
      );
      break;
```